frontend/src/slices/auth/actions.ts

```diff
@@ -6,13 +6,13 @@ import {
   type UserSignUpResponseDto,
 } from '~/packages/users/users.js';
 
-import { name as sliceName } from './auth.slice.js';
+import { AuthActionType } from './common.js';
 
 const signUp = createAsyncThunk<
   UserSignUpResponseDto,
   UserSignUpRequestDto,
   AsyncThunkConfig
->(`${sliceName}/sign-up`, (registerPayload, { extra }) => {
+>(AuthActionType.SIGNUP, (registerPayload, { extra }) => {
```

Comment:
Approach with enum for action types may lead to unexpected misleads (in case if someone would use action-type name, but sliceName would be different)
```suggestion
>(`${sliceName}/sign-up`, (registerPayload, { extra }) => {
```